config/samples/dataplane_v1beta1_openstackdataplanenodeset_bgp.yaml

```diff
@@ -33,24 +33,10 @@ spec:
             storage_ip: 172.18.0.100
             tenant_ip: 172.19.0.100
             fqdn_internal_api: edpm-compute-0.example.com
-  networkAttachments:
-    - ctlplane
```

Comment:
The kubebuilder marker that sets this default wasn't actually having any effect since it was on AnsibleEESpec. I've attempted to clean that up in https://github.com/openstack-k8s-operators/dataplane-operator/pull/591